src/Microsoft.Tye.Hosting/DockerRunner.cs

```diff
@@ -291,7 +311,23 @@ async Task RunDockerContainer(IEnumerable<(int ExternalPort, int Port, int? Cont
                     }
                 }
 
-                var command = $"run -d {workingDirectory} {volumes} {environmentArguments} {portString} --name {replica} --restart=unless-stopped {dockerImage} {docker.Args ?? ""}";
+                // When running on the host network, make service names resolve to 127.0.0.1.
+                // When running on a specific network, service names are resolved via aliases passed to the 'network connect' command.
+                string hostNetworkArgs = "";
+                if (application.UseHostNetwork)
+                {
+                    hostNetworkArgs = $"--network host";
+                    foreach (var serviceName in localhostServices!)
+                    {
+                        hostNetworkArgs += $" --add-host {serviceName}:127.0.0.1";
+                    }
+                }
+
+                // Workaround podman issue: https://github.com/containers/libpod/issues/6508
+                // Fixed in podman v2.
+                bool isPodman = await DockerDetector.Instance.IsPodman.Value;
+                string restartArg = isPodman ? "always" : "unless-stopped";
```

Comment:
What does this mean for shutdown?

Reply:
This affect what happens when the system restarts.
For "docker", "unless-stopped" means only containers which weren't stopped will be started at boot.
"podman" doesn't start containers at boot. With podman v2, "always" and "unless-stopped" are aliases.